op-program/client/boot/boot_interop.go

```diff
@@ -38,6 +39,7 @@ type OracleConfigSource struct {
 
 	customConfigsLoaded bool
 
+	l1ChainConfigs map[eth.ChainID]*params.ChainConfig
```

Comment:
I think this should just be a singleton l1ChainConfig - I don't see how we would support multiple l2 chains connected to different l1's. 

```suggestion
	l1ChainConfig *params.ChainConfig
```

We should probably validate that this is the case somewhere.

Reply:
I think that's right, there needs to be single L1 chain so that we can unify the liquidty https://specs.optimism.io/interop/eth-lockbox.html